python/simulation/fake_raft.py

```diff
@@ -10,14 +10,31 @@
 
 from __future__ import print_function, absolute_import, division
 
+import sys
 import os
 import fnmatch
 import yaml
 
 import astropy.io.fits as fits
+
+import siteUtils
 from datacat.error import DcClientException
 from DataCatalog import DataCatalog
-# import siteUtils
+
+
+# Configure the database interface
+ROOT_FOLDER = 'LSST/mirror/SLAC-prod/prod'
+ETRAV_DB = 'Dev'
+USER = os.environ['USER']
+
+# Defaults for writing fake files
+OUTPATH = '.'
+
+# These should come from the Job Harness
+JOB_ID = siteUtils.getJobName()
```

Comment:
If these aren't used later in this file, I'd just delete them and use the `siteUtils` functions explicitly in the calling code instead of these constants.